sgx_main.c

```diff
@@ -74,14 +74,22 @@
 #define DRV_DESCRIPTION "Intel SGX Driver"
 #define DRV_VERSION "2.6.0"
 
+#ifndef MSR_IA32_FEAT_CTL
+#define MSR_IA32_FEAT_CTL MSR_IA32_FEATURE_CONTROL
+#endif
+
+#ifndef FEAT_CTL_LOCKED
+#define FEAT_CTL_LOCKED FEATURE_CONTROL_LOCKED
+#endif
+
 MODULE_DESCRIPTION(DRV_DESCRIPTION);
 MODULE_AUTHOR("Jarkko Sakkinen <jarkko.sakkinen@linux.intel.com>");
 MODULE_VERSION(DRV_VERSION);
 #ifndef X86_FEATURE_SGX
 	#define X86_FEATURE_SGX (9 * 32 + 2)
 #endif
 
-#define FEATURE_CONTROL_SGX_ENABLE                      (1<<18)
+#define FEAT_CTL_SGX_ENABLE                      (1<<18)
 
```

Comment:
new name is FEAT_CTL_SGX_ENABLED?
Also need wrap with ifndef.

Reply:
Are you asking me to change the name of this macro? Because I didn't name it this.

I'm confused what the ifndef is for. This macro is declared here for the first time.

Reply:
Fixed.